src/main/webapp/js/user-page.js

```diff
@@ -304,6 +314,11 @@ function init() {
         configureButtons(user);
         configureUserProfile(user);
       });
+  fetch('/api/user-post-url')
+      .then((response) => response.text())
+      .then((url) => {
+        setProfileFormUrl(url);
+      });
```

Comment:
``` 
$.ajax('/api/user-post-url').done((url) => {
    setProfileFormUrl(url);
  });
```
just a suggestion, I thought jquery looks shorter, but you can keep the fetch if you want